src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -52,12 +52,21 @@ public ResponseEntity<?> deleteLine(@PathVariable Long id) {
     }
 
     @PostMapping("/{id}/sections")
-    public ResponseEntity<SectionResponse> addSection( @PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
+    public ResponseEntity<SectionResponse> addSection(@PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
         SectionResponse sectionResponse = lineService.saveSection(id, sectionRequest);
-        return ResponseEntity.created(URI.create("/lines/"+id+"/sections")).body(sectionResponse);
+        return ResponseEntity.created(URI.create("/lines/" + id + "/sections")).body(sectionResponse);
     }
 
-    @ExceptionHandler(value = {EntityNotFoundException.class, IllegalArgumentException.class })
+    @DeleteMapping("/{lineId}/sections")
+    public ResponseEntity<?> removeLineStation(
```

Comment:
추상화를 위한 의도가 아니라면  와일드카드보다는 타입을 명확하게 명시해주거나 해당 경우에는 아예 제거 하는게 나을 것 같아요
```suggestion
    public ResponseEntity removeLineStation(
```